packages/jellyfish-api-core/__tests__/category/oracle/listPrices.test.ts

```diff
@@ -0,0 +1,100 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+
+describe('Oracle', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeEach(async () => {
```

Comment:
```suggestion
  beforeAll(async () => {
```

Reply:
I really need beforeEach. If not, the 2nd test will fail.

Reply:
does this mean `removeoracle` does not affect `listPrices` RPC? Your data will still be there when you call `listprice`?

Reply:
Hahaha. I just figure out what you mean.

Reply:
If you see other oracle tests that can be written like this, you can optimize them moving ahead.